lib/generator.js

```diff
@@ -1,38 +1,159 @@
 'use strict';
 
+const common = require('metarhia-common');
+
+const structures = [
+  'boolean',
+  'null',
+  'undefined',
+  'number',
+  'string',
+  'symbol',
+];
+
+const gObjects = [
+  'Object',
+  'Array',
+  'Date',
+  'Function',
+  'Map',
+  'WeakMap',
+  'Set',
+  'WeakSet',
+  'Int8Array',
+  'Uint8Array',
+  'Uint8ClampedArray',
+  'Float32Array',
+  'Error',
+  'EvalError',
+  'TypeError',
+  'RangeError',
+  'SyntaxError',
+  'ReferenceError',
+];
+
+const types = [
+  ...structures,
+  ...gObjects,
+  'Primitive',
+];
+
+const baseUrl = 'https://developer.mozilla.org/en-US/docs/';
+const structuresBaseUrl = 'Web/JavaScript/Data_structures#';
+const gObjectsBaseUrl = 'Web/JavaScript/Reference/Global_Objects/';
+
+const structuresLinks = structures.map(type =>
+  [type, `${baseUrl}${structuresBaseUrl}${common.capitalize(type)}_type`]
+);
+
+const gObjectsLinks = gObjects.map(type =>
+  [type, `${baseUrl}${gObjectsBaseUrl}${type}`]
+);
+
+let links = [
+  ...gObjectsLinks,
+  ...structuresLinks,
+  [ 'Primitive', baseUrl + 'Glossary/Primitive' ],
+];
+
+links = links.map(link => '[`<' + link[0] + '>`]: ' + link[1]);
+
+const wrapType = type => {
+  if (types.includes(type)) {
+    return '[`<' + type + '>`][]';
+  } else if (type.startsWith('[')) {
+    type = type.slice(1, type.length - 1)
+      .replace(/ /, '')
+      .split(',')
+      .map(s => s.slice(1, s.length - 1))
```

Comment:
This will probably fail for smth like `[[<A>, <B>], <C>]`. Or we won't support such cases?

Reply:
I think we can support this kind of syntax.